src/models/chapter/start-time-offset.yaml

```
@@ -0,0 +1,8 @@
+type: object
+required:
+  - startTimeOffset
+description: "The start time offset for the chapter"
+properties:
+  startTimeOffset:
+    type: integer
+    example: 0 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)